### PR TITLE
Add option to modify existing users with nipap-passwd

### PR DESCRIPTION
**Pull request by plajjan**

### CODE REVIEW

File: nipap/nipap-passwd.man.rst [hunk 1]
```diff
@@ -29,6 +34,10 @@ Options
 	delete
     	Delete user with username specifies with **-u** from the database
 
+    modify
```
Comment (garberg) [inline nipap/nipap-passwd.man.rst]: There seems to be a mixture of tabs & space for indentation here. I guess space is what we want and thus the added entry is correct, but perhaps you could fix the other entries in the list of arguments while we're at it..?

**Comment by garberg** [comment]
Looks good to me, just added a note regarding the indentation being inconsistent in the manual file. The branch is out-of-date so I cannot merge, fix that (and the indentation if you wish, otherwise we can take care of that some other time) and I'll merge!

**Comment by plajjan** [comment]
Ok, fixed indentation and some other small thingy in the man page (was missing *).

**Comment by garberg** [comment]
Nice, merging!

**Pull request merged**